src/main/java/frc/robot/Robot.java

```diff
@@ -180,26 +237,31 @@ public void robotInit() {
         // Set the resolution
         camera.setResolution(Constants.CAMERA_WIDTH, Constants.CAMERA_HEIGHT);
 
-        // Get a CvSink. This will capture Mats from the camera
-        CvSink cvSink = CameraServer.getVideo();
+        SmartDashboard.putBoolean("Estimated Controller Status", true);
 
-        InputtedCoDriverControls.setCoDriverController(coDriverController);
-        InputtedDriverControls.setDriverController(driverController);
```

Comment:
We now get the controllers statically from the Robot class instead of our previous silly method.